dotrix_core/src/assets/animation.rs

```diff
@@ -4,7 +4,8 @@ use super::{
     skin::JointId,
 };
 
-use crate::math::{ TransformBuilder, slerp };
+use super::super::renderer::transform::{ TransformBuilder };
```

Comment:
also renamed legacy math into transform and moved into renderer, just not sure about this line...